src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +18,36 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(actor);
+            transaction.commit();
+        } catch (HibernateException e) {
+            throw new DataProcessingException("Cannot open the session", e);
```

Comment:
The exception message 'Cannot open the session' might be misleading since the exception is caught after the session is opened. Consider changing it to something more general like 'Error during transaction' to cover all possible Hibernate exceptions.